builder/builder.go

```diff
@@ -768,12 +778,17 @@ func (b *Builder) AddBundles(bundles []string, allLocal bool, allUpstream bool,
 		upstreamBundleDir := getUpstreamBundlesPath(b.UpstreamVer)
 		upstreamSet, err := b.getDirBundlesListAsSet(upstreamBundleDir)
 		if err != nil {
-			return errors.Wrapf(err, "Failed to read upstream bundles dir: %s", upstreamBundleDir)
+			return errors.Wrapf(err, "Failed to read upstream bundles dir: %q", upstreamBundleDir)
```

Comment:
these `%s`->`%q` changes don't seem related

Reply:
That's fair. In general I want to normalize output (and have a separate patch I'm doing right now that changes over the output for the rest of the commands to be consistent with each other), but I can make these changes there if you'd prefer.

Reply:
I prefer commits to be a single functional or logical change. This is a rule I often mess up myself, but there you go... At least for small commits like this it is easy to manage.

Reply:
I totally agree, I just often get caught up in trying to reduce future work at the expense of breaking this rule. So here, I knew I wanted to convert them all to `%q` in my other patch, so lazily tried to get it over with while I was editing that section.

But again, totally agree, and it's something I need to be way stricter about.

Reply:
Took out all these changes. :)